service_sysv_linux.go

```diff
@@ -177,7 +177,8 @@ const sysvScript = `#!/bin/sh
 
 cmd='{{.Path}}{{range .Arguments}} {{.}}{{end}}'
 
-name=$(basename $0)
+real_path=$(readlink -f $0)
+name=$(basename $real_path)
```

Comment:
So it's the same as upstream change: https://github.com/percona/kardianos-service/pull/2/commits/03b7b20ce7aa394d80c159de55c7ec2c69f3849c
Were we able to reproduce the problem or is this commit just a backport?